app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java

```diff
@@ -152,6 +155,12 @@ public void updateFromItem(final InfoItem infoItem,
             itemPublishedTime.setText(item.getTextualUploadDate());
         }
 
+        if (item.isHeartedByUploader()) {
+            itemHeartView.setVisibility(View.VISIBLE);
+        } else {
+            itemHeartView.setVisibility(View.GONE);
+        }
+
```

Comment:
I just realized this should be moved to `CommentsInfoItemHolder`, a subclass of `CommentsMiniInfoItemHolder`, since the `list_comments_mini_item.xml` layout does not contain the heart, only `list_comments_item.xml` does